gradle/maven-push.gradle

```diff
@@ -92,21 +77,47 @@ afterEvaluate { project ->
         sign configurations.archives
     }
 
-    //task androidJavadocs(type: Javadoc) {
-    //source = android.sourceSets.main.allJava
-    //}
+    task androidJavadocs(type: Javadoc) {
+        source = android.sourceSets.main.java.srcDirs
+        ext.androidJar = "${android.sdkDirectory}/platforms/${android.compileSdkVersion}/android.jar"
+        classpath += files(ext.androidJar)
+    }
 
-    //task androidJavadocsJar(type: Jar, dependsOn: androidJavadocs) {
-    //classifier = 'javadoc'
-    //from androidJavadocs.destinationDir
-    //}
+    task androidJavadocsJar(type: Jar, dependsOn: androidJavadocs) {
+        classifier = "javadoc"
+        from androidJavadocs.destinationDir
+    }
 
     task androidSourcesJar(type: Jar) {
-        classifier = 'sources'
+        classifier = "sources"
         from android.sourceSets.main.java.sourceFiles
     }
 
+    if (JavaVersion.current().isJava8Compatible()) {
+        allprojects {
+            tasks.withType(Javadoc) {
+                options.addStringOption("Xdoclint:none", "-quiet")
+            }
+        }
+    }
+
     artifacts {
         archives androidSourcesJar
+        archives androidJavadocsJar
+    }
+
+    task installArchives(type: Upload) {
```

Comment:
To test out `tinydancer` locally, you can run ./gradlew installArchives to publish to your local maven folder: `.m2`.